S4TF_Tutorial_3.ipynb

```diff
@@ -406,7 +406,7 @@
         "## Reading the data\n",
         "\n",
         "We need to read the file into an array of bytes which can further be passed to `readMNIST` function to convert it into a tensor array. \\\n",
-        "The value of each pixel in the image data is an integer in the range `[0,255]`. For the model to work properly, these values need to be normalized to the range `[0,1]`. So here we normalize the image by dividing the pixel value by 255, and then apply it to each image in the test and train datasets. \\\n",
+        "The value of each pixel in the image data is an integer in the range `[0, 255]`. For the model to work properly, these values need to be normalized to the range `[0, 1]`. So here we normalize the image by dividing the pixel value by 255, and then apply it to each image in the test and train datasets. \\\n",
```

Comment:
The line breaks at the end of each of these lines got messed up somehow, turning them into slashes. All of the lines in this paragraph should have the linebreaks corrected at the end of them.